app/views/vehicles_adm.py

```diff
@@ -0,0 +1,127 @@
+from flask import render_template, request, Blueprint, flash, redirect, abort, url_for
+from flask_login import login_required
+
+from app import app, util
+import app.models.vehicles as vehicle_model
+
+
+bp = Blueprint("vehicles", __name__)
+
+
+@bp.route("/admin/vehicles")
+@login_required
+def admin_vehicles_management():
+    vehicles = vehicle_model.all()
+    return render_template(
+        "admin/vehicles/index.html", vehicles=vehicles, active_nav="vehicle"
+    )
+
+
+@bp.route("/admin/vehicles/save", methods=["POST"])
+@login_required
+def admin_vehicles_save():
+    uploaded_images = None
+    if request.files.getlist("images"):
+        uploaded_images = util.upload_files(
```

Comment:
I wonder what would happen to the files if two files have the same name? I can see a situation where images stored and named as 1.png or image1.png or car1.png and so on? 
is it possible to rename the images before storing them? we could use the random_id generator and we could pass a letter "i" as its initial? 

What do you think?

Reply:
Yeah I have thought about this too. How about we take the record id? It already has the initial letter + random_id.
Do that sound good to you?

Reply:
sounds good to me

Reply:
On the second thought. Your idea is better because it requires to change only one place :D